DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/BulkUploadController.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates
+{
+    using System;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Helpers;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.FeatureManagement.Mvc;
+
+    [FeatureGate(FeatureFlags.RefactoredTrackingSystem)]
+    [Authorize(Policy = CustomPolicies.UserCentreAdmin)]
+    [Route("TrackingSystem/Delegates/BulkUpload")]
+    public class BulkUploadController : Controller
+    {
+        private readonly IDelegateDownloadFileService delegateDownloadFileService;
+
+        public BulkUploadController(IDelegateDownloadFileService delegateDownloadFileService)
+        {
+            this.delegateDownloadFileService = delegateDownloadFileService;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        [Route("DownloadDelegates")]
+        public IActionResult DownloadDelegates()
+        {
+            var content = delegateDownloadFileService.GetDelegateDownloadFileForCentre(User.GetCentreId());
+            return File(
+                content,
+                "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+                $"ITSP Delegates for Bulk Update {DateTime.Today:yyyy-MM-dd}.xlsx"

```

Comment:
Kevin came back to me about this - could we name it "DLS Delegates...." and add this to the JIRA? 🙏